include/psa/crypto_struct.h

```diff
@@ -159,6 +159,8 @@ struct psa_aead_operation_s
      * any driver (i.e. none of the driver contexts are active). */
     unsigned int MBEDTLS_PRIVATE(id);
 
+    uint64_t MBEDTLS_PRIVATE(associated_key_id);
+    mbedtls_svc_key_id_t MBEDTLS_PRIVATE(key_id);
```

Comment:
When `MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER` is active, this is a two-element structure (owner and id).